core/client/app/templates/components/gh-profile-image.hbs

```diff
@@ -16,5 +16,5 @@
             <span class="sr-only">Upload an image</span>
         </i>
     </span>
-    <input type="file" class="file-uploader js-file-input" name="uploadimage">
+    {{#if fileStorage}}<input type="file" class="file-uploader js-file-input" name="uploadimage">{{/if}}
```

Comment:
fileStorage is named in the config so it sounds more reasonable to keep that consistent, I think this flag is used elsewhere as well iirc